using-timescaledb/continuous-aggregates.md

```diff
@@ -18,118 +18,127 @@ re-calculate all of the data on every refresh. Only new and/or
        AVG(temperature),
        MAX(temperature),
        MIN(temperature)
 FROM conditions
 GROUP BY device, bucket;
 ```
 
-A `time_bucket` on the time partitioning column of the hypertable is required in all continuous aggregate views. If you do
-not provide one, you will get an error. The `time_bucket` function in
-this case has a bucket width of 1 hour.
+Lastly, you should add a policy to ensure that the continuous
+aggregate is refreshed on a regular basis.
+
+```sql
+SELECT add_continuous_aggregate_policy('conditions_summary_daily',
+	start_interval => INTERVAL '1 month',
+	end_interval => INTERVAL '1 h',
+	schedule_interval => INTERVAL '1 h');
+```
+
+In this case, the continuous aggregate will be refreshed every hour
+and refresh the last month's data.
 
-You can create multiple continuous aggregates on the same
-hypertable. For example, you can create a continuous aggregate
-`condition_summary_daily` over the data with daily buckets:
+You can now run a normal `SELECT` on the continuous aggregate and it
+will give you the aggregated data, for example, to select the averages
+for the first three months:
 
 ```sql
-CREATE VIEW conditions_summary_daily
+SELECT bucket, avg FROM conditions_summary_daily
+WHERE device = 1 AND bucket BETWEEN '2020-01-01' AND '2020-03-31'
+ORDER BY bucket;
+```
+
+### A detailed look at continuous aggregate views [](detailed-look)
+
+As you saw above, creating a refreshing [continuous
+aggregate][api-continuous-aggs] consists of creating a continuous
+aggregate view of the data using [`CREATE MATERIALIZED
+VIEW`][postgres-createview] with the `timescaledb.continuous` and
+adding a continuous aggregate policy for the continuous aggregate to
+keep it refreshed.
+
+You can create several continuous aggregates for the same
+hypertable. For example: to create a continuous aggregate view is
+updated hourly, you would write:
+
+```sql
+CREATE MATERIALIZED VIEW conditions_summary_hourly
 WITH (timescaledb.continuous) AS
 SELECT device,
-       time_bucket(INTERVAL '1 day', time) AS bucket,
+       time_bucket(INTERVAL '1 hour', time) AS bucket,
        AVG(temperature),
        MAX(temperature),
        MIN(temperature)
 FROM conditions
 GROUP BY device, bucket;
+
+-- Create the policy
+SELECT add_continuous_aggregate_policy('conditions_summary_hourly',
+	start_interval => INTERVAL '1 month',
+	end_interval => NULL,
+	schedule_interval => INTERVAL '5 min');
 ```
 
+A `time_bucket` on the time partitioning column of the hypertable is
+required in all continuous aggregate views. If you do not provide one,
+you will get an error.
+
+When the view is created, it will (by default) be refreshed so that it
+contain an aggregate of the current data in `conditions`. If you make
+changes to the hypertable and want to refresh an interval, you can
+explicity refresh it by calling
+[`refresh_continuous_aggregate`][refresh_continuous_aggregate] and
+give the range you want to refresh:
```

Comment:
Should note what a refresh means: we are not rebuilding from scratch. Regions that were invalidated will be rebuilt. But other regions are not touched. So it is efficient. (cc @erimatnor please comment if this is correct).

Reply:
Added a comment about that.

Reply:
@gayyappan it's correct.